backend/requirements.txt

```diff
@@ -1,10 +1,12 @@
-uvicorn==0.23.2
-fastapi==0.109.1
+uvicorn
+fastapi
 alembic==1.12.0
 psycopg2==2.9.9
 asyncpg==0.28.0
 sqlalchemy[asyncio]==2.0.20
 PyPDF2==3.0.1
 pgvector==0.3.5
 sentence-transformers==3.2.0
-python-multipart==0.0.12
+python-multipart
+chainlit==1.3.2
```

Comment:
We don't need `chainlit` for the backend. Suggest moving it to a requirements.txt within chainlit

Reply:
Yes, it should be here. I moved it because it couldn't reference correctly, and this was done during a debugging session with Mark

Reply:
Removed